app/init.php

```diff
@@ -178,13 +179,13 @@ function($value) {
 $register->set('cache', function () { // Register cache connection
     $redis = new Redis();
     $redis->pconnect(App::getEnv('_APP_REDIS_HOST', ''), App::getEnv('_APP_REDIS_PORT', ''));
-    $auth = App::getEnv('_APP_REDIS_AUTH',null);
-    if($auth != null) {
-        $auth = explode(':',$auth);
-        if(!empty(trim($auth[0]))) {
-            $redis->auth($auth);
+    $user = App::getEnv('_APP_REDIS_USER','') ?? '';
+    $pass = App::getEnv('_APP_REDIS_PASS','') ?? '';
+    if(!empty($pass)) {
```

Comment:
We should allow authentication if user or password have been set.

No need for `??` check. `getEnv` already takes care of that internally and return a default value (second param).

Reply:
updated